proto/cosmos/accounts/v1/query.proto

```diff
@@ -23,3 +27,40 @@ message AccountQueryResponse {
   // response defines the query response of the account.
   bytes response = 1;
 }
+
+// SchemaResponse is the response type for the Query/Schema RPC method.
+message SchemaRequest {
```

Comment:
these are approximate schema definitions which represent the init/query/exec messages an account can process.